guides/en/1_getting_started/3_components.md

```diff
@@ -118,8 +121,11 @@ player.use(sprite("big-butterfly"));
 player.use("big");
 ```
 
-Use is a **use**ful method, but what if you want to remove a component/tag? You
-can use the `GameObjRaw.unuse()` method, passing the component id or tag name.
+*(Aside: the `.use()` method is actually how `add()` creates the object internally. It initializes an empty `GameObj` and then calls `.use()` on all of the components and tags passed in. Don't just take my word for it, [look at the code!](https://github.com/kaplayjs/kaplay/blob/b57385be0f33b89db79ced6a4967d219d240f523/src/game/make.ts#L624-L626))*
+
+
+`use()` is a **use**ful method, but what if you want to remove a component or tag? You
+can use the `GameObjRaw.unuse()` method, passing the component ID or tag name. Do note that you can't pass the component itself to `unuse()`, you have to pass the component *name*!
```

Comment:
The Do note that you can pass the component itself can be in a extra note, as the previous thing. Will let you know then, for now, remove